frontend/src/metabase/visualizations/visualizations/CartesianChart/chart-definition.ts

```diff
@@ -97,3 +112,16 @@ export const getCartesianChartDefinition = (
     ...props,
   };
 };
+
+export const COMBO_CHARTS_SETTINGS_DEFINITIONS = {
+  ...LINE_SETTINGS,
+  ...GRAPH_GOAL_SETTINGS,
+  ...GRAPH_TREND_SETTINGS,
+  ...GRAPH_COLORS_SETTINGS,
+  ...GRAPH_AXIS_SETTINGS,
+  ...GRAPH_DISPLAY_VALUES_SETTINGS,
+  ...GRAPH_DATA_SETTINGS,
+  ...TOOLTIP_SETTINGS,
+  ...STACKABLE_SETTINGS,
+  ...LEGEND_SETTINGS,
+} as any as VisualizationSettingsDefinitions;
```

Comment:
Stack settings have now dropped to the bottom of the list and should probably still remain at the top, since they are more visualization specific.

**Master**
![image](https://github.com/metabase/metabase/assets/22608765/82e3d688-cc50-4f0a-af7b-cf8f881bf7ee)


**This PR**
![image](https://github.com/metabase/metabase/assets/22608765/54b0f221-e01f-4aed-9072-459993a4994f)


```suggestion
export const COMBO_CHARTS_SETTINGS_DEFINITIONS = {
  ...LINE_SETTINGS,
  ...STACKABLE_SETTINGS,
  ...GRAPH_GOAL_SETTINGS,
  ...GRAPH_TREND_SETTINGS,
  ...GRAPH_COLORS_SETTINGS,
  ...GRAPH_AXIS_SETTINGS,
  ...GRAPH_DISPLAY_VALUES_SETTINGS,
  ...GRAPH_DATA_SETTINGS,
  ...TOOLTIP_SETTINGS,
  ...LEGEND_SETTINGS,
} as any as VisualizationSettingsDefinitions;
```